packages/terra-clinical-result/src/flowsheet-result-cell/FlowsheetResultCell.jsx

```diff
@@ -141,12 +141,8 @@ const createClinicalResultDisplay = (children, hasUnverifiedIcon, hasInterpretat
   return (<div key={(`ClinicalResultDisplay-${resultKeyID}`)} className={primaryResultClassnames} ref={containerDivRef}>{children}</div>);
 };
 
-const createStandardResultDisplay = (resultDataItem, hasUnverifiedIcon, hasInterpretationIcon, hideUnit, resultKeyID, numericOverflow, containerDivRef) => {
-  const {
-    status,
-  } = resultDataItem;
+const createStandardResultDisplay = (resultDataItem, isStatusInError, hasUnverifiedIcon, hasInterpretationIcon, hideUnit, resultKeyID, numericOverflow, containerDivRef) => {
```

Comment:
which one of these params are required and which ones are optional? taking a page out of matt's book, could we move towards an object param api for these functions?

Reply:
I think we might want to save bigger API changes in the cleanup PR that is eventually coming.  I've dropped it down to just passing in the attributes object for now. https://github.com/cerner/terra-clinical/pull/639/commits/2aa1cc5bf73202c44d9808132ec1015e621fa52c

Reply:
what is planned for the cleanup pr? do we need to create an issue that outlines these changes?

Reply:
Currently the cleanup PR is just going through the original release PR for clinical-result https://github.com/cerner/terra-clinical/pull/574 and addressing all the comments and issues on it that didn't get implemented along with going through the code to see if there are code blocks that could be clearer and easier to read. I don't think a tracking issue would hurt, since this isn't so much a bug as an internal enhancement.